ui/imports/shared/controls/AssetsDetailsHeader.qml

```diff
@@ -0,0 +1,107 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+
+import utils 1.0
+import shared.controls 1.0
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+
+Item {
```

Comment:
Since this is a new component... Start with `Control` at the toplevel, and use a proper layout for `contentItem` instead of these anchors, will make the code _much_ cleaner

Reply:
Hey, iv used it and addressed other review comments please have a look at it when you can :)